### PR TITLE
[airship-sdk-10.0.2] update to airship SDK 10.0.2

### PR DESCRIPTION
**Pull request by marc-scig**
Hi there,

This updates the airship android SDK 10.0.2. Some things have changed as of 10.x requiring minor code changes, namely:

* Replacement of AirshipReceiver with listener interfaces
* Minor refactor of the CustomEvent builder to bring it in line with other builders in the SDK

We don't have a shared team account on mParticle so I'm unable to test this beyond verifying that the project compiles. I'd appreciate some help verifying it. Thanks.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -26,6 +26,7 @@ android {
 }
 
 dependencies {
+    api 'androidx.annotation:annotation:1.1.0'
```
Comment (marc-scig) [inline build.gradle]: Note: I'm not sure if this is necessary in the normal way you folks build this project, but all the nonnull/nullable annotations in the source were problematic for me until I added this dependency.

Comment (rlepinski) [reply build.gradle]: Should not be needed. This project is not using androidx yet.

Comment (willpassidomo) [reply build.gradle]: Hey guys, the androidx usage was inadvertently pushed into the master branch and has been reverted. If you rebase onto the current master, you will be able to remove the androidx dependency. Sorry for the confusion

Comment (marc-scig) [reply build.gradle]: Fixed

Comment (willpassidomo) [reply build.gradle]: Awesome, thanks you. I will merge this right away, and my dangling androidx commit. As always, we appreciate the PR, and this should be released next Wednesday.

**Pull request merged**